src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -522,6 +522,46 @@ ss::future<response_ptr> describe_configs_handler::handle(
               request.data.include_synonyms,
               &describe_as_string<model::timestamp_type>);
 
+            // Shadow indexing properties
+            add_topic_config_if_requested(
+              resource,
+              result,
+              topic_property_remote_read,
+              false,
+              topic_property_remote_read,
+              (topic_config->properties.shadow_indexing
+               == model::shadow_indexing_mode::full)
+                  || (topic_config->properties.shadow_indexing == model::shadow_indexing_mode::fetch)
+                ? std::make_optional(true)
+                : std::make_optional(false),
+              request.data.include_synonyms,
+              &describe_as_string<bool>);
+
+            add_topic_config_if_requested(
+              resource,
+              result,
+              topic_property_remote_write,
+              false,
+              topic_property_remote_write,
+              (topic_config->properties.shadow_indexing
+               == model::shadow_indexing_mode::full)
+                  || (topic_config->properties.shadow_indexing == model::shadow_indexing_mode::archival)
+                ? std::make_optional(true)
+                : std::make_optional(false),
+              request.data.include_synonyms,
+              &describe_as_string<bool>);
+
+            add_topic_config_if_requested(
```

Comment:
I'm not sure that we should show `redpanda.remote.recovery` flag. This is actually a product level decision. If we will be showing the flag it will be there all the time after recovery. On a bright side, it will be possible to tell which topics are recovered and which are not. We should probably ask @sameers361 and @vsaraswat what to do.

Reply:
@twmb commented in [slack](https://vectorizedio.slack.com/archives/C02BDN76HUK/p1638996482361900?thread_ts=1637615453.257400&cid=C02BDN76HUK) that only `redpanda.remote.{read,write}` should be displayed. Updated commit